sources/main.cpp

```diff
@@ -25,12 +26,7 @@ main(int argc, char* argv[]) {
     }
     dfl::common::Log::init(options);
 
-    std::stringstream ss;
-    ss << "Hello World: ";
-    for (int i = 0; i < argc; i++) {
-      ss << argv[i] << " ";
-    }
-    LOG(info) << ss.str() << LOG_ENDL;
+    dfl::Context context(options.config().networkFilePath);
```

Comment:
No unit test? will they come later?

Reply:
Context is not meant to have a unit test as it will be no different from launching the main. But network manager will have a unit test in future pull request